cypress/features/regression/message.feature

```diff
@@ -65,14 +64,13 @@ Feature: Message component
 
   @positive
   Scenario: Enable on dismiss state for a Message component
```

Comment:
I think this should be split to two scenarios

Reply:
fixed